### PR TITLE
homework_lesson-5

### PR DESCRIPTION
**Pull request by DenisKishylia**
Finalization with fonts and creation of layout footer

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -30,6 +30,64 @@ <h1 class="offer_title">What to be offer</h1>
...
+                        <a href="#">Contact Customer Service</a>
+                    </li>
+                    <li>
+                        <a href="#">FAQ</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="footer__info">
+            &copy; 2020 Triphouse, Inc. All rights reserved
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:89]: The text should wrap tag `p, span, a and similar`.

File: Triphouse-project/style.css [hunk 2]
```diff
@@ -50,4 +79,54 @@ height: 10px;
...
+    padding: 20px;
+  }
+  li {
+    height: 50px;
+    font-family: "Roboto";
+    font-size: medium;
+    color: #ffff;
+  }
+
+  li:before {
```
Comment (BubentsovKirill) [inline Triphouse-project/style.css:117]: Use margin for `li` item instead `before`.

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -30,6 +30,64 @@ <h1 class="offer_title">What to be offer</h1>
                 </div>
             </div>
         </div>
-    </section>
+    </main>
+    <footer class="footer">
+        <div class="footer-content">
+            <div class="footer-content__list">
```
[the remark this answers is not in the document]
Comment (DenisKishylia) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html [hunk 4]
```diff
@@ -30,6 +30,64 @@ <h1 class="offer_title">What to be offer</h1>
...
+                        </li>
+                        <li>
+                            <a href="#">Privacy</a>
+                        </li>
+                        <li>
+                            <a href="#">Terms</a>
+                        </li>
+                </ul>
+            </div>
+            <div class="footer-content__list">
```
[the remark this answers is not in the document]
Comment (DenisKishylia) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html [hunk 5]
```diff
@@ -30,6 +30,64 @@ <h1 class="offer_title">What to be offer</h1>
...
+                    </li>
+                    <li>
+                        <a href="#">Holiday homes</a>
+                    </li>
+                    <li>
+                        <a href="#">Hostes</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="footer-content__list">
```
[the remark this answers is not in the document]
Comment (DenisKishylia) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html (hunk 1 above)
Comment (DenisKishylia) [reply Triphouse-project/index.html]: fixed. Adding text to <p>

File: Triphouse-project/style.css (hunk 2 above)
Comment (DenisKishylia) [reply Triphouse-project/style.css]: fixed

**Pull request merged**